src/pages/WithdrawCryptoPage/index.tsx

```diff
@@ -0,0 +1,364 @@
+import * as classnames from 'classnames';
+import {Field, FieldProps, Form, Formik, FormikProps} from 'formik';
+import {computed} from 'mobx';
+import {inject, observer} from 'mobx-react';
+import * as React from 'react';
+import {Link, RouteComponentProps} from 'react-router-dom';
+import Yup from 'yup';
+import {RootStoreProps} from '../../App';
+import {AmountInput} from '../../components/AmountInput';
+import {Banner} from '../../components/Banner';
+import {
+  ROUTE_CONFIRM_OPERATION_ID,
+  ROUTE_SECURITY,
+  ROUTE_WITHDRAW_CRYPTO_FAIL
+} from '../../constants/routes';
+import {STORE_ROOT} from '../../constants/stores';
+import {BalanceModel, OpStatus, WithdrawCryptoModel} from '../../models';
+import {moneyRound} from '../../utils';
+
+import './style.css';
+
+interface WithdrawCryptoPageProps
+  extends RootStoreProps,
+    RouteComponentProps<any> {}
+
+export class WithdrawCryptoPage extends React.Component<
+  WithdrawCryptoPageProps
+> {
+  readonly assetStore = this.props.rootStore!.assetStore;
+  readonly withdrawStore = this.props.rootStore!.withdrawStore;
+  readonly walletStore = this.props.rootStore!.walletStore;
+  readonly profileStore = this.props.rootStore!.profileStore;
+  readonly socketStore = this.props.rootStore!.socketStore;
+
+  @computed
+  get balance() {
+    const {assetId} = this.props.match.params;
+
+    if (this.walletStore.tradingWallets.length) {
+      const balance = this.walletStore.tradingWallets[0].balances.find(
+        (assetBalance: BalanceModel) => {
+          return assetBalance.assetId === assetId;
+        }
+      );
+
+      if (balance) {
+        return balance.balance;
+      }
+    }
+
+    return 0;
+  }
+
+  componentDidMount() {
+    const {assetId} = this.props.match.params;
+    this.withdrawStore.fetchFee(assetId);
```

Comment:
Should it be await this.widthdrawStore... ?

Reply:
We can't await inside `componentDidMount` :(